great_expectations/checkpoint/actions.py

```diff
@@ -1131,6 +1131,8 @@ def __init__(
         :param site_names: *optional* List of site names for building data docs
         """
         super().__init__(data_context)
+        if isinstance(site_names, str):
+            site_names = [site_names]
```

Comment:
This is a runtime change

Reply:
I've reverted these changes for now.
@alexsherstinsky has a story in the backlog to overhaul `checkpoint`